components/chat/doc-list.tsx

```diff
@@ -0,0 +1,57 @@
+import { IDocument } from '@/types/knowledge';
```

Comment:
don't need shows all documents, just display the document use upload in the chat.